src/templates/options.js

```diff
@@ -22,9 +22,8 @@ for (const [key, value] of Object.entries(options)) {
 
 <% if (options.langDir) { %>
 export const ASYNC_LOCALES = {
-  <%= options.locales
-    .map(l => `'${l.file}': () => import('../${relativeToBuild(options.langDir, l.file)}' /* webpackChunkName: "lang-${l.file}" */)`)
-    .filter((l, index, self) => self.indexOf(l) === index)
-    .join(',\n  ') %>
+  <%= Array.from(
+        new Set(options.locales.map(l => `'${l.file}': () => import('../${relativeToBuild(options.langDir, l.file)}' /* webpackChunkName: "lang-${l.file}" */)`))
```

Comment:
Cool! I was thinking to unique on and then generate imports but this is also nice pattern might :D